src/components/forums/forum/CreateForum.tsx

```diff
@@ -187,6 +188,7 @@ export function CreateForum(props: CreateForumProps) {
           ),
           type: MessageType.success,
         });
+        process.env.NEXT_PUBLIC_SUPABASE_URL && collectionPublicKey && addForum(collectionPublicKey, title);
```

Comment:
is there a reason that this variable would not be declared?
if so, then lets error out the server in the very beginning.

basically, if theres a variable that needs to be defined for a critical op, then we shouldn't silently fail - the person starting the server shouldn't be able to start it at all before they fix the var.

Reply:
it likely won't be defined when used by another customer (ex. somebody integrating into their own website), we could include the keys in the package, but then they would be publicly exposed. something to consider is if we want to track forums created on other websites, and if so how.